cmake/nlohmann_jsonConfig.cmake

```diff
@@ -0,0 +1,20 @@
+#
```

Comment:
what's this stuff all about, and isn't it going to find the system nlohmann? We definitely don't want that to happen.